app/services/announcement_notification_router.rb

```diff
@@ -0,0 +1,14 @@
+class AnnouncementNotificationRouter
```

Comment:
There's not much too this right now but down the road this will end up getting a lot more complex, thus the separate service object.